components/Login.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <div className="font-bold w-[694px] h-[564px] m-10">
+      <form className=" bg-acm-black rounded-[57px] grid">
```

Comment:
Instead of `grid` as a CSS element we are going to use Bootstrap Grid instead. Here is a good example of how to get it working, https://github.com/acm-ucr/acm-hydra/blob/dev/components/Upcoming.js#L23-L36 

The setup is already done for bootstrap, you will just have to implement it